frontend/app-development/features/processEditor/handlers/OnProcessTaskRemoveHandler.test.ts

```diff
@@ -54,14 +112,68 @@ describe('OnProcessTaskRemoveHandler', () => {
       } as TaskEvent,
     };
 
-    const onProcessTaskRemoveHandler = new OnProcessTaskRemoveHandler(
-      org,
-      app,
+    const onProcessTaskRemoveHandler = createOnRemoveProcessTaskHandler({
       currentPolicy,
-      mutateApplicationPolicy,
-    );
+    });
     onProcessTaskRemoveHandler.handleOnProcessTaskRemove(taskMetadata);
 
-    expect(mutateApplicationPolicy).toHaveBeenCalledWith(expectedResponse);
+    expect(mutateApplicationPolicyMock).toHaveBeenCalledWith(expectedResponse);
```

Comment:
should we add a `toHaveBeenCalledTimes(1)` as well to see that it is not called too many times?